Shoko.Server/nlog.config

```diff
@@ -13,7 +13,13 @@
     <logger name="Shoko.Server.API.Authentication.CustomAuthHandler" maxlevel="Info"
             final="true"/> <!-- Auth log spam (blackhole) -->
 
-    <logger name="*" minlevel="Info" writeTo="file"/>
+    <logger name="*" minlevel="Info" writeTo="file">
```

Comment:
Have you tested this? I'm not sure how some nlog.config stuff works with our code overrides in https://github.com/ShokoAnime/ShokoServer/blob/73542e3fb10c64b2f05e76c9c6e93c2a70e373af/Shoko.Server/Utilities/Utils.cs#L75

Reply:
I've given it a test and it works for file based logging (in line with previous comments)...

Realistically, given that troubleshooting can be performed using `server-settings.json`, I'm not entirely sure why I only bothered implementing this for the file target (especially if I was proposing removing the censorship added the original way).

I've now implemented the previous changes in `Utils.cs` with the above filter rules applying to the file log target, and additionally the SignalR & console log targets. Will get this pushed onto the branch shortly.